source/guide_satis.rst

```diff
@@ -0,0 +1,116 @@
+.. author:: Pascal Iske <info@pascal-iske.de>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/composer-satis.png
+      :align: center
+
+##########
+Composer Satis
+##########
+
+To manage private composer packages you can either use a private packagist account or host it for free by yourself with composer satis.
+The latter is as easy as cloning the official composer satis repository and editing the config file.
+Satis will then build a static html site with your packages.
+This guide shows you how to quickly setup satis on your Uberspace account.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Composer_
```

Comment:
Broken link

Reply:
Fixed the broken link and added another link to the new manual.